src/test/tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "esModuleInterop": true,
     "module": "commonjs",
     "strictNullChecks": true,
-    "types": ["jest", "codeceptjs"]
+    "types": ["jest", "codeceptjs","node"],
```

Comment:
cery minor issue...missing space between "codeceptjs" and "node" after the comma

Reply:
Fixed